rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2020-05-15
+stable-2020-10-08
```

Comment:
if it is stable, why do we even need this file

Reply:
It's the one we use for `near-sdk-rs`. Maybe people had old toolchains by default in the past.